scripts/config.rb

```diff
@@ -5,18 +5,16 @@
 require 'erb'
 require 'ostruct'
 
-TEMPLATE = "/root/filebeat.yml.erb"
-CONFIG = "/root/config.yml"
+TEMPLATE = ENV["FILEBEAT_TEMPLATE"] || "/root/filebeat.yml.erb"
```

Comment:
yup. I built an image yongtin/logzio-filebeat to verify this. 

Ran the config.rb and generated the same /filebeat.yml file with the same md5 hash as logzio/logzio-filebeat. Main diff now is I can customize the path if I wanted to via environment variables.